advanced/contributing-docs.md

```diff
@@ -7,39 +7,43 @@ description: Contribute to TinyMCE's developer documentation.
+
+For our Contributor Code of Conduct, visit our [Contributor Covenant Code of Conduct on GitHub](https://github.com/tinymce/tinymce-docs/blob/develop/CODE_OF_CONDUCT.md#contributor-covenant-code-of-conduct).
 
 ## How to contribute to TinyMCE's documentation
 
 **TL;DR** A quick way to improve the docs is to click the "Was this article helpful?" and select "no". Then if you want to help more click the "opening an issue" link and submit a GitHub issue.
 
-### So, how can you help?
+## So, how can you help?
 
-We welcome people with technical writing experience to help make TinyMCE's docs as useful to our community as possible. You don't need to be an expert developer to contribute, but a basic understanding of software development is helpful.
+We welcome people with technical writing experience to help make TinyMCE's docs as useful to our community as possible. Although you don't need to be an expert developer to contribute, a basic understanding of software development is helpful.
 
-What is more important is your ability to articulate complex ideas. Our goal is to help developers understand TinyMCE functionality as quickly as possible so that they can build great stuff.
+What is more important is your ability to articulate complex ideas. Our goal is to help developers understand TinyMCE's functionality as quickly as possible.
 
-If that sounds like something you are into, branch the [docs GitHub repository](https://github.com/tinymce/tinymce-docs) and submit a PR, or reach out to the [TinyMCE docs team](mailto:marketing@tiny.cloud) if you have a specific question.
+Simply branch the [docs GitHub repository](https://github.com/tinymce/tinymce-docs) and submit a PR, or reach out to the [TinyMCE docs team](https://github.com/tinymce/tinymce-docs-4x/issues/new?assignees=&labels=question&template=question.md) if you have a specific question.
 
-### TinyMCE docs tech stack
+## TinyMCE documentation tech stack
 
-The documentation is built with Jekyll. Text files are written in [Markdown](https://help.github.com/articles/markdown-basics/) and published on [GitHub](https://github.com/tinymce/tinymce-docs). You'll find the [repo here](https://github.com/tinymce/tinymce-docs).
+The documentation is built with Jekyll. Text files are written in [Markdown](https://help.github.com/articles/markdown-basics/) and published on [GitHub](https://github.com/tinymce/tinymce-docs). You will find the [repo here](https://github.com/tinymce/tinymce-docs).
 
-### Markdown & code syntax
+## Markdown & code syntax
 
-As we mentioned above, we write using [GitHub Flavored Markdown](https://help.github.com/articles/github-flavored-markdown/). If you're not familiar with Markdown [read this](https://help.github.com/articles/markdown-basics/).
+The documentation is written in [GitHub Flavored Markdown](https://help.github.com/articles/github-flavored-markdown/). If you are not familiar with Markdown [read this](https://help.github.com/articles/markdown-basics/).
```

Comment:
I know it was like this, but I'm pretty sure it this is wrong. The flavours are likely very similar though, so maybe these were just cleaner docs?